packages/commerce-sdk-react/src/hooks/ShopperLogin/useShopperLoginHelpers.ts

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright (c) 2022, Salesforce, Inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+import {ActionResponse, ShopperLoginInstance} from '../types'
+import {ShopperLoginHelpers} from './types'
+
+// The useShopperLoginHelper hook builds on top of the
+// SLAS helpers from commerce-sdk-isomorphic. The actions
+// have one to one relationship with the helpers from
+// commerce-sdk-isomorphic.
+//
+// On top of sending the HTTP response, these helpers will be
+// responsible for managing tokens.
```

Comment:
Do these helpers need to be the ones managing the token? What if something else (another class) does it? For example: https://github.com/SalesforceCommerceCloud/pwa-kit/pull/666#discussion_r921641966